### PR TITLE
Xs 3641

### PR DESCRIPTION
**Pull request by allenbross-wf**
### Changes:

- Added new resource files for DQC.US.0001 and DQC.US.0015
- Updated the pin to reflect new release

### Pull request process

- [ ] Full description of changes provided above.
- [ ] Reviewed by another person. (+1)
- [ ] If this is a code change, reviewed by a second person. (+1)
- [ ] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

**Comment by brianmyers-wf** [comment]
Looks good to me, but build failed in testing?

**Comment by allenbross-wf** [comment]
Check in 8dc70c4 should fix the CI issue.  The tests that had been run were for a different version of the rules.

File: dqc_us_rules/resources/DQC_US_0001/dqc_0001.json [hunk 1]
```diff
@@ -339,7 +353,13 @@
 			"SICAD1Member", 
 			"SICAD2Member", 
 			"OtherCurrencyMember", 
-			"NonUSDollarMember"
+			"NonUSDollarMember",
+		    "CENCOEXMember",
```
Comment (allenbross-wf) [inline dqc_us_rules/resources/DQC_US_0001/dqc_0001.json]: I fixed this with commit: 10444ac1322da0cd0f026e6dfad83aa13ab56e97
It's included in this PR but comes after this commit.

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by brianmyers-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+10 - passes test cases.  Merging.

**Comment by davidtauriello** [comment]
@andrewperkins-wf @allenbross-wf @jasonleinbach-wf – PR #203 should also be
reviewed and merged to next_v3_16 before prerelease v3.0.0RC1 is created,
as it contains related documentation for public exposure.



David





David Tauriello
*Vice President, Operations*

(202) 448-1985



*xbrl.us* <http://xbrl.us/> is the national consortium

for the business reporting standard.



*From:* Jay Leinbach [mailto:notifications@github.com]
*Sent:* Tuesday, October 11, 2016 11:54 AM
*To:* DataQualityCommittee/dqc_us_rules
*Subject:* Re: [DataQualityCommittee/dqc_us_rules] Xs 3641 (#204)



+10 - passes test cases. Merging.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/DataQualityCommittee/dqc_us_rules/pull/204#issuecomment-252959723>,
or mute the thread

**Pull request merged**